### PR TITLE
Disabling experimental features

### PR DESCRIPTION
**Pull request by piotrpekala7**

### CODE REVIEW

File: src/app/services/settings.service.ts [hunk 1]
```diff
@@ -57,10 +55,6 @@ export class SettingsService {
     });
   }
 
-  isExperimentalEnabled(): boolean {
```
Comment (piotrpekala7) [inline src/app/services/settings.service.ts]: @ziajka  Did you mean not removing options in the settings service?

**Comment by ziajka** [comment]
yes, can be useful in the future

**Comment by piotrpekala7** [comment]
ready for merging

File: src/app/components/settings/settings.component.spec.ts [hunk 2]
```diff
@@ -48,7 +48,6 @@ describe('SettingsComponent', () => {
   it('should get and save new settings', () => {
     const settings = {
       crash_reports: true,
-      experimental_features: true,
```
Comment (ziajka) [inline src/app/components/settings/settings.component.spec.ts]: We may want to keep it

File: src/app/services/settings.service.spec.ts [hunk 3]
```diff
@@ -6,9 +6,6 @@ import { Settings, SettingsService } from './settings.service';
 export class MockedSettingsService {
   settings = {};
 
-  isExperimentalEnabled() {
```
Comment (ziajka) [inline src/app/services/settings.service.spec.ts]: We may want to keep it

File: src/app/services/settings.service.spec.ts [hunk 4]
```diff
@@ -59,7 +56,7 @@ describe('SettingsService', () => {
   it('should get all values', inject([SettingsService], (service: SettingsService) => {
     expect(service.getAll()).toEqual({
       crash_reports: true,
-      experimental_features: false,
```
Comment (ziajka) [inline src/app/services/settings.service.spec.ts]: We may want to keep it

File: src/app/services/settings.service.spec.ts [hunk 5]
```diff
@@ -73,7 +70,7 @@ describe('SettingsService', () => {
 
     expect(service.getAll()).toEqual({
       crash_reports: false,
-      experimental_features: false,
```
Comment (ziajka) [inline src/app/services/settings.service.spec.ts]: We may want to keep it

File: src/app/services/settings.service.spec.ts [hunk 6]
```diff
@@ -93,10 +90,4 @@ describe('SettingsService', () => {
       expect(changedSettings.crash_reports).toEqual(false);
     })
   ));
-
-  it('should get isExperimentalEnabled when turned on', inject([SettingsService], (service: SettingsService) => {
```
Comment (ziajka) [inline src/app/services/settings.service.spec.ts]: We may want to keep it

File: src/app/services/settings.service.ts [hunk 7]
```diff
@@ -4,7 +4,7 @@ import { BehaviorSubject } from 'rxjs';
 
 export interface Settings {
   crash_reports: boolean;
-  experimental_features: boolean;
```
Comment (ziajka) [inline src/app/services/settings.service.ts]: We may want to keep it

File: src/app/services/settings.service.ts [hunk 8]
```diff
@@ -13,7 +13,7 @@ export interface Settings {
 export class SettingsService {
   static DEFAULTS: Settings = {
     crash_reports: true,
-    experimental_features: false,
```
Comment (ziajka) [inline src/app/services/settings.service.ts]: We may want to keep it

File: src/app/services/settings.service.ts (hunk 1 above)
Comment (ziajka) [reply src/app/services/settings.service.ts]: We may want to keep it

**Comment by piotrpekala7** [comment]
It's easier to open new pr than to revert changes

**Pull request closed without merging**